plasmapy/plasma/grids.py

```diff
@@ -355,6 +357,20 @@ def unit(self):
     # *************************************************************************
     # 1D axes and step sizes (valid only for uniform grids)
     # *************************************************************************
+    @property
+    def si_scale_factors(self):
+        """
+        Scale factors that convert the units of each axis to SI units.
+
+        """
```

Comment:
```suggestion
    def si_scale_factors(self) -> List[float]:
        """
        3-element list containing unitless scale factors for converting
        the corresponding axis from its stored units to SI.
        """
```

You'll also need to import `List` from `typing`.

Reply:
Darn! I was too slow in removing the old comment and making this one.  I'll redo this comment.

Reply:
nevermind